apps/coscrad-frontend/src/utils/generic-components/style-components/coscrad-main-content-container.tsx

```diff
@@ -0,0 +1,21 @@
+import { Container } from '@mui/material';
+import { ReactNode } from 'react';
+import { FloatSpacerDiv } from '../float-spacer';
+
+interface CoscradMainContentContainerProps {
+    children: ReactNode;
+}
+
+export const CoscradMainContentContainer = ({
+    children,
+}: CoscradMainContentContainerProps): JSX.Element => {
+    return (
+        <Container
+            component="div"
+            sx={{ marginTop: '5% !important', width: { xs: '90%', sm: '80%' }, margin: 'auto' }}
```

Comment:
Not a big fan of `!important`. Is there another way to make this take precedence?

Reply:
Replaced it with padding.  This is where CSS is hard to work with because of the layer of control that react and webpack have over stylesheets.  Hard to target a div directly without a straight stylesheet.  We may need more research into MUI layout to get this right.